services/sts/src/main/java/software/amazon/awssdk/services/sts/auth/StsAssumeRoleCredentialsProvider.java

```diff
@@ -117,5 +117,13 @@ public Builder refreshRequest(Supplier<AssumeRoleRequest> assumeRoleRequestSuppl
         public Builder refreshRequest(Consumer<AssumeRoleRequest.Builder> assumeRoleRequest) {
             return refreshRequest(AssumeRoleRequest.builder().applyMutation(assumeRoleRequest).build());
         }
+
+        /**
+         * {@inheritDoc}
+         */
+        public StsAssumeRoleCredentialsProvider build() {
```

Comment:
@Override? Also, inheritDoc is the default behavior, so we shouldn't need to say it explicitly.